### PR TITLE
EOY Appeal

### PR DESCRIPTION
**Pull request by micahilbery**
First pass at the EOY Appeal. I'm really struggling with our build system today. So it's fully not where I want it to be but this is a first pass. I think the typography could be improved as well as some animations and things.

### CODE REVIEW

**Comment by Alecaddd** [comment]
It's a good start, but of course it's missing a lot of things.
If it can help, I was able to solve all the problems by manually installing python 2.7 on macOS.
https://www.python.org/ftp/python/2.7.18/python-2.7.18-macosx10.9.pkg

With that, I'm able to use the regular `pip` to install the requirements and run the watch command without problems.
Shutdown and restart are fast.

The only thing I had to change was removing [this chunk](https://github.com/thundernest/thunderbird-website/blob/master/website/_includes/bedrock/base-resp.html#L83) from the bedrock base file, otherwise it wouldn't compile for me.

**Comment by ryanleesipes** [comment]
@micahilbery @Alecaddd have you guys synced up on what is missing? I'd like to get this merged today. We still have translations to do.

**Comment by micahilbery** [comment]
I have it working on static local files because I can't get the build system to work. I have copied that work into the live files so it should work in theory but I cant really test it.

File: website/thunderbird/102.0/eoy/index.html
Comment (Alecaddd) [inline website/thunderbird/102.0/eoy/index.html:56]: missing donation URL.
@ryanleesipes which donation link + utm we want here?

File: website/thunderbird/102.0/eoy/index.html
Comment (Alecaddd) [inline website/thunderbird/102.0/eoy/index.html:102]: also this

**Review by Alecaddd — CHANGES_REQUESTED** [review]
Why the hard line on the background image? It looks nothing like the designed page.

The style of the page needs more polish and refinement, but the text is fine (after the few issues addressed), so if it's good for @ryanleesipes and @Sancus, we can merge it fo the purpose of translations, and then continue the improvements without touching the strings in a follow up PR

File: website/thunderbird/102.0/eoy/index.html
Comment (Alecaddd) [reply website/thunderbird/102.0/eoy/index.html:56]: This is the link we should add here: https://give.thunderbird.net/?utm_source=thunderbird-client&utm_medium=referral&utm_content=EOY22

File: website/thunderbird/102.0/eoy/index.html
Comment (Alecaddd) [reply website/thunderbird/102.0/eoy/index.html:27]: I spoke with Andrei and indeed this would not be understandable or usable for a lot of languages.

Also I would suggest to style the whole H1 with the larger font we need for the "Thunderbird Alive" part, and then a simple span without class to style the exception, which is the smaller and thinner font.

We need to have something like this, I think:
```
{{ _('<span>Help Keep</span>Thunderbird Alive!')}}
```

We can also avoid the `<br>` by making that span a display block.

File: media/css/eoy.css
Comment (Alecaddd) [inline media/css/eoy.css:102]: no need to repeat the same values for dark mode, these should be maintained from the previous body style.

**Review by Alecaddd — CHANGES_REQUESTED** [review]

File: media/css/eoy.css
Comment (Alecaddd) [inline media/css/eoy.css:313]:
```suggestion
  inset: -4px;
```

**Review by Alecaddd — APPROVED** [review]
This is good to land for the strings.
@ryanleesipes and @Sancus, waiting for your final review and merging decision.

**Comment by ryanleesipes** [comment]
LGTM. Let's merge @Sancus

**Review by ryanleesipes — APPROVED** [review]
Haven't loaded up the site (reviewing from phone). But, from what I can tell - this looks good to go

**Pull request merged**
